### ISSUE TITLE
relation to RDFAuth

### ISSUE THREAD

**Issue by bblfish** [w3c-ccg/http-signatures#12]
I just noticed that I had written up a post in 2008 on a sketch of a protocol that looks very much like
Http Signature :-)

https://web.archive.org/web/20090306221745/http://blogs.sun.com:80/bblfish/entry/rdfauth_sketch_of_a_buzzword

I then discovered that TLS could do this out of the box with UI support, so I left the `WWW-Authenticate` road behind.

**Comment by liamdennehy** [comment]
@bblfish Do we still want to address this?

**Comment by bblfish** [comment]
Not sure how to address this, just thought I'd point it out.

**Comment by liamdennehy** [comment]
I think this specification plugs into that proposal as an authentication means, so could have utility. Considering it's not a (proposed) standard I don't think we can mention it directly.

I did however have a curious conversation where someone claimed this protocol is more secure than Mutual TLS in the instance where intermediaries are involved. Once I pointed out this protocol has little in the way of replay protection his argument fell apart. That said, your proposed challenge/response mechanism (the nonce) has value, though I don't know how/whether to incorporate it. I am looking at an overhaul of how we specify headers, and a nonce in the signature header itself as a parameter could be appropriate.

I'll close this for now, if you have a specific proposal feel free to make a new issue.
